js-array-methods-2_animal-strings/index.js

```diff
@@ -21,15 +21,17 @@ const animalStrings = [
   "rhino",
 ];
 
-const hippoExists = null;
+const hippoExists = animalStrings.includes("hippo");
 
 // Hint: You can pass the starting index as second parameter.
-const catStartingFromIndexFiveExists = null;
+const catStartingFromIndexFiveExists = animalStrings.includes("cat", 5);
 
 // Hint: Besides the array method, check out the string method `startsWith()`.
-const firstAnimalStartingWithLetterP = null;
+const firstAnimalStartingWithLetterP = animalStrings.find((animal) =>
+  animal.startsWith("p")
+);
 
-const indexOfGiraffe = null;
+const indexOfGiraffe = animalStrings.findIndex((animal) => animal == "giraffe");
```

Comment:
why `==` and not `===`?

To go a little further, you don't know how the string is gonna be.. Giraffe, GiRaffe, giraffe, giraFFe. What method can you think of that will help you solve this problem